hermes-management/src/main/java/pl/allegro/tech/hermes/management/infrastructure/metrics/MetricsQuery.java

```diff
@@ -0,0 +1,4 @@
+package pl.allegro.tech.hermes.management.infrastructure.metrics;
+
+public record MetricsQuery(String query) {
```

Comment:
why not using string?

Reply:
I wanted to somewhat make the query independent of a specific provider, but now I don't see any benefits so I can delete it.

Reply:
fixed